### DISCUSSION TITLE
How to pass the user's real ip address when proxying the request

### DISCUSSION THREAD

**Discussion by ymh199478** [nitrojs/nitro#1379] [Q&A]
I am using nitro in unxt3，Made the following configuration:

```
routeRules: {
    '/api/**': {
         proxy: 'http://' + process.env.NUXT_BACKEND_ENDPOINT + '/**'
    }
}
```

How to carry `X-Real-IP` header when forwarding

**Comment by ahoiroman** [comment]
Did you ever solve that? I am still trying to figure that out.
